gridworld/visualizer.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, render_size=(64, 64)) -> None:
         self.renderer = Renderer(self.world, self.agent,
             width=self.render_size[0], height=self.render_size[1],
             caption='Pyglet', resizable=False)
-        self.renderer.overlay = True
```

Comment:
What's this?

Reply:
This is a 2d debug info like player's coord, inventory, aim point, etc